WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -45,7 +46,11 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		Observable.FromEventPattern<string>(_qrReader, nameof(_qrReader.InvalidAddressFound))
 			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(args => Message = $"Invalid QR code.")
+			.Subscribe(args =>
+			{
+				ErrorMessage = "The QR code is not a valid Bitcoin address";
```

Comment:
I'm not sure the message is great. 

What about: *No valid Bitcoin address found/detected.* Would it be better or not?